packages/next-dashboard/src/utils/useMutationObserver.js

```diff
@@ -14,9 +14,17 @@ export default function useMutationObserver(
   target: ?Node,
   options: MutationObserverInit | void | null,
   callback: ?Callback,
+  inputs: ?$ReadOnlyArray<mixed>,
 ) {
   if (typeof window === 'undefined') return;
-  const [observer] = useState(new MutationObserver(gcb));
+  const observer = useMemo(() => new MutationObserver(gcb));
+  useEffect(() => {
+    return () => {
+      if (observer) {
+        callbacks.delete(observer);
+      }
```

Comment:
Why is this conditional required? Could this ever be falsy? (I mean, *in theory* a constructor function can return `null` I believe, but I think it's fair to assume `new MutationObserver` never does...)